tf-modules/archive/async_operation.tf

```diff
@@ -17,9 +17,13 @@ resource "aws_ecs_task_definition" "async_operation" {
       {
         "name": "AWS_REGION",
         "value": "${data.aws_region.current.name}"
+      },
+      {
+        "name": "databaseCredentialSecretArn",
+        "value": "${var.rds_user_access_secret_arn}"
       }
     ],
-    "image": "cumuluss/async-operation:27",
+    "image": "jlkovarik/ecs-task-runner:38",
```

Comment:
Because the two new packages aren't published, and Docker won't let you bring symlinked lerna deps in, this requires a manual build until the release.

Reply:
Yes - I think we should get the PR approved, pass int tests, manually rebuild and then re-pass the integration tests with the updated image. 

I'll leave that up to the team as to if it should be a seperate PR.

Reply:
If integration tests pass without being too painful, I'd probably vote on doing it in this PR, but we can see what happens